src/wdlci/wdl_tests/compare_array_array_strings.wdl

```diff
@@ -0,0 +1,51 @@
+version 1.0
+
+# Check if two arrays of arrays of strings are identical by comparing the flattened arrays
+# Input type: Array[Array[String]]
+
+task validate_and_compare_pbsv_splits {
```

Comment:
Tasks in this repo should be generic and apply to any outputs of the given type - this may be a typo, but the task here should be named the same as the file itself